js/sdk/src/components/BundlrModal.js

```diff
@@ -0,0 +1,276 @@
+import React, { useState, useEffect, useContext, useMemo } from 'react'
+import { styled } from '@mui/material/styles'
```

Comment:
if `BundlrModal` is being rendered from the SDK, it should be removed from hubs and web